fdupes.1

```diff
@@ -42,6 +42,9 @@ omit the first file in each set of matches
 .B -A --nohidden
 exclude hidden files from consideration
 .TP
+.B -x --xdev
+skip direcories on different filesystems
```

Comment:
Should be "directories".

Reply:
Not that it makes a difference, since this project seems pretty dead :disappointed:

Reply:
Thanks, good catch. Yes it's disappointing, but jdupes seems to fill the gap.